### ISSUE TITLE
[OpenBMC] Come up with easier ways to debug/validate xCAT response vs REST response.

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#3728]
I'm opening this issue as we have internal customers bringing up issues with commands ...(https://github.com/xcat2/xcat-core/issues/3726) regarding OpenBMC ... rpower, rvitals, rinv and xCAT pass through data from the REST api calls it's difficult to ask them to get data with us because we have to format a curl command for them to run.  

If we ask the customer to turn `xcatdebugmode=1` .. .and rerun... we get the request data, but we don't get any response data...

Then we have to convert the command to curl format and ask the customer to run to make sure the data returned from REST is actually correct.   

Here's what we get with `xcatdebugmode=1` 
```
[root@stratton01 ~]# rvitals frame6u05
frame6u05: [openbmc_debug] POST https: //10.6.5.253/login -d { "data": [ "root", "0penBmc" ] }
frame6u05: [openbmc_debug] login_response 200 OK
frame6u05: [openbmc_debug] GET https: //10.6.5.253/xyz/openbmc_project/sensors/enumerate
frame6u05: [openbmc_debug] rvitals_response 200 OK
frame6u05: [openbmc_debug] Processing command: rvitals all
frame6u05: Ambient: 22.101 C
frame6u05: Fan0 0: 9765 RPMS
frame6u05: Fan0 1: 10330 RPMS
frame6u05: Fan1 0: 9842 RPMS
frame6u05: Fan1 1: 10330 RPMS
frame6u05: Fan2 0: 9791 RPMS
frame6u05: Fan2 1: 10273 RPMS
frame6u05: Fan3 0: 9765 RPMS
frame6u05: Fan3 1: 10330 RPMS
frame6u05: P0 Vcs Temp: 27 C
frame6u05: P0 Vdd Temp: 26 C
frame6u05: P0 Vddr Temp: 26 C
frame6u05: P0 Vdn Temp: 26 C
frame6u05: P1 Vcs Temp: 26 C
frame6u05: P1 Vdd Temp: 26 C
frame6u05: P1 Vddr Temp: 27 C
frame6u05: P1 Vdn Temp: 27 C
```

However, if the value of fan speed is 100,000, while we do have the endpoint, we can't directly drop this into a curl command and show the data...  

If we do have the curl data, @hu-weihua , test testcase could do REAL validation on the data that is returned from the REST call and what xCAT is displaying in the command, or detect if we are missing data, etc....  

Open for any ideas on how we can improve this process, or perhaps log the data somewhere if debugmode=1 ?

**Comment by hu-weihua** [comment]
@whowutwut . You are right, test case should do more useful check. It make sense to invoke REST directly in test case to valid the data that is returned from the REST call and what xCAT is displaying in the command. @junxiawang, could you update your cases to do it? thanks.

**Comment by gurevichmark** [comment]
Could it be as simple as just printing out the response data we get back ?

**Comment by whowutwut** [comment]
Yes, but I don't think `Dumper` works in Hierarchy... but I think we want to make sure it only gets called when `xcatdebugmode=1` so that we don't affect normal performance of xCAT commands.

**Comment by whowutwut** [comment]
Looks like @xuweibj Opened #3796 and this is resolved by #3813
